src/main/java/at/tugraz/sw20asd/lang/ui/Controller/OverviewVocabs.java

```diff
@@ -0,0 +1,68 @@
+package at.tugraz.sw20asd.lang.ui.Controller;
+
+import at.tugraz.sw20asd.lang.model.Vocabulary;
+import at.tugraz.sw20asd.lang.ui.VocabularyAccess;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.Button;
+
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Font;
+
+import java.io.*;
+
+public class OverviewVocabs extends VBox {
+
+    private VocabularyAccess vocab;
+
+    @FXML
+    private Button return_btn;
+
+    public OverviewVocabs(VocabularyAccess vocab) {
+        this.vocab = vocab;
+        FXMLLoader loader = new FXMLLoader();
+        loader.setControllerFactory(c -> this);
+        loader.setRoot(this);
+        try {
+            loader.load(getClass().getResource("/overview-vocabs.fxml").openStream());
+
+        } catch (IOException ex) {
+            throw new RuntimeException(ex);
+        }
+    }
+
+    public void initialize() {
+
+        for (Vocabulary v : vocab.getAllVocabularies()) {
```

Comment:
This implementation is okay for a few vocabularies, but if you have a larg database, the gui will freeze or in worst case you have to force quit the computer. Therefore, the preferred option is to implement JavaFX threads called Tasks or Services. In your case I would recommend Tasks. 
Therefor it would make sense if the command getAllVocabularies doesn't return a collection but a list of Vocabularies. `private void getVocabs(){

        getAllVocabsTask = new Task<>() {
            @Override
            protected Collection<Vocabulary> call() throws Exception {
                Vocabularies = vocab.getAllVocabularies();
                Platform.runLater(()-> {
                    System.out.println(Vocabularies);
                });
                return Vocabularies;
            }
        };

        getAllVocabsTask.stateProperty().addListener(((observable, oldValue, newValue) -> {
            if(newValue == Worker.State.CANCELLED){
                //think about which feedback could be useful for the user
                //changes on the gui are only allowed via the JavaFX main Thread
                //therefore you have to use Platform.runLater() otherwise you get nullptr exceptions
                getAllVocabsTask.cancel();
            }

            if(newValue == Worker.State.SUCCEEDED){
                if(Vocabularies == null || Vocabularies.size() == 0){
                    Platform.runLater(()-> {
                        //some useful  message
                    });
                }
            }
        } ));

        //this is how you start the thread and return the resources at the end of the task
        // (after he has fetched the vocabulary)
        Thread th = new Thread(getAllVocabsTask);
        th.setDaemon(true);
        th.start();
    }`

If you want to use this, change the  return value of the  command to List<Vocabulary> and adapt the code above.